src/main/java/pl/byczazagroda/trackexpensesappbackend/service/FinancialTransactionServiceImpl.java

```diff
@@ -91,8 +91,8 @@ public FinancialTransactionDTO updateTransaction(
                 });
 
         financialTransaction.builder()
```

Comment:
@RobertoJavaDev please refactor this code to this version:
```java
FinancailTransaction ft = FinancialTransaction.builder()
                .amount(updateTransactionDTO.amount())
                .description(updateTransactionDTO.description())
                .transactionDate(Instant.now())
                .build();

return financialTransactionModelMapper.mapFinancialTransactionEntityToFinancialTransactionDTO(ft);
```

Reply:
**UPDATE:**
Instead of `.builder()` pattern please just use setters on object `FinancailTransaction ft`.
Thank you very much. Sorry for my little mistake.

Reply:
Corrected